src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/auth/signin/SignInPage.java

```diff
@@ -0,0 +1,23 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.auth.signin;
+
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.FormPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.forgotpassword.ForgotPasswordPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.register.RegisterPage;
+
+public interface SignInPage extends FormPage {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move constants to a class or enum. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1214)